src/wobjectdefs.h

```diff
@@ -234,6 +234,23 @@ constexpr auto addString(const StaticStringList<T> &l, const StaticString<L> & s
     return binary::tree_append(l, s);
 }
 
+template<class T>
+struct W_TypeWrap {};
```

Comment:
maybe you can use identity_t (define earlier)?

Reply:
identity_t is gone now, so i think this needs to stay?